demoapp/src/androidTest/java/me/uport/sdk/demoapp/VerifyJWTTest.kt

```diff
@@ -15,14 +16,19 @@ import me.uport.sdk.ethrdid.EthrDIDResolver
 import me.uport.sdk.jsonrpc.JsonRPC
 import me.uport.sdk.universaldid.UniversalDID
 import org.hamcrest.CoreMatchers.not
+import org.hamcrest.Matchers.allOf
 import org.junit.Rule
 import org.junit.Test
+import ro.mirceanistor.testutil.ViewMatcherIdlingRule
 
 class VerifyJWTTest {
 
     @get:Rule
     val activityRule = ActivityTestRule(VerifyJWTActivity::class.java)
 
+    @get:Rule
+    val viewMatcherIdlingRule = ViewMatcherIdlingRule(allOf(withId(R.id.progress), isDisplayed()))
+
```

Comment:
`viewMatcherIdlingRule` doesn't seem to ever be used. Is it necessary to have a variable for this?